EIPS/eip-6900.md

```diff
@@ -0,0 +1,358 @@
+---
+eip: 6900
+title: Modular Smart Contract Accounts and Plugins
+description: An interface for modular smart contract accounts with plugins.
```

Comment:
The description restates too much of the title. You also have several interfaces, not just one.

How about something like:

```suggestion
description: Interfaces for composable plugins optionally supporting upgradability and introspection 
```

Reply:
Updated to "Interfaces for composable account plugins optionally supporting upgradability and introspection"